life_expectancy/cleaning.py

```diff
@@ -2,21 +2,35 @@
 import argparse
 import pandas as pd
 
-def clean_data(region: str) -> None:
-    """ Clean the data for a given region.
+def main(region: str) -> None:
+    """Main function."""
+    full_data = _load_data('life_expectancy/data')
+    region_data = _clean_data(full_data, region)
+    _save_data_csv('life_expectancy/data', region_data, region)
+
+def _clean_data(data: pd.DataFrame, region: str) -> pd.DataFrame:
+    """ Clean the data and return only a given region.
 
     Args:
         region: the name of the region to clean
     """
-    data_path = 'life_expectancy/data'
     full_data = (
-        pd.read_table(f'{data_path}/eu_life_expectancy_raw.tsv')
+        data
         .pipe(_melt_table)
         .pipe(_clean_year_value_cols)
         .pipe(_clean_region_col)
     )
     region_data = _select_region(full_data, region)
-    _save_data_csv(data_path, region_data, region)
+
+    return region_data
+
+def _load_data(data_path: str) -> pd.DataFrame:
+    """Load the data.
+
+    Returns:
+        data (pd.DataFrame): The data.
+    """
+    return pd.read_table(f'{data_path}/eu_life_expectancy_raw.tsv')
 
```

Comment:
QUESTION: In the _clean_region_col function, you are splitting the region names. What is the reason to do this? My impression is that the regions include not just individual countries, but also EU and EEA at various points in time and therefore have the years included in that case.

Reply:
I have that doubt also, so i dont know if what I did is correct... Again, lets discuss this in the meeting :)